### PR TITLE
telegram-bot

### PR DESCRIPTION
**Pull request by Hat1ko**
creation of telegram bot

### CODE REVIEW

**Comment by Hat1ko** [comment]
Here i went for break

File: wrapper-ripple/src/main/java/com/hatiko/ripple/database/service/XrpDatabaseOperatorImpl.java [hunk 1]
```diff
@@ -144,19 +145,35 @@ public Boolean deleteUserByUsername(String username, String password) {
...
+		
+		AccountLastIdEntity entityToDelete = accountLastIdRepo.findOneByPublicKey(publicKey);
+		
+		if(Optional.ofNullable(entityToDelete).isEmpty()) {
+			return Boolean.TRUE;
+		}
+		
+		accountLastIdRepo.delete(entityToDelete);
+		
+		return Optional.ofNullable(accountLastIdRepo.findOneByPublicKey(publicKey)).isEmpty();
```
Comment (SophiToyber) [inline wrapper-ripple/src/main/java/com/hatiko/ripple/database/service/XrpDatabaseOperatorImpl.java]: Why didn't you use the ternary operator?

Comment (Hat1ko) [reply wrapper-ripple/src/main/java/com/hatiko/ripple/database/service/XrpDatabaseOperatorImpl.java]: logic : i search for element -> if i dont find one -> i return true -> else : delete and search again

File: wrapper-ripple/src/main/java/com/hatiko/ripple/WrapperRippleApplication.java [hunk 2]
```diff
@@ -2,12 +2,17 @@
 
 import org.springframework.boot.SpringApplication;
 import org.springframework.boot.autoconfigure.SpringBootApplication;
-import org.springframework.scheduling.annotation.EnableScheduling;
+import org.telegram.telegrambots.ApiContextInitializer;
 
 //@EnableScheduling
```
Comment (SophiToyber) [inline wrapper-ripple/src/main/java/com/hatiko/ripple/WrapperRippleApplication.java]: why "//"?

Comment (Hat1ko) [reply wrapper-ripple/src/main/java/com/hatiko/ripple/WrapperRippleApplication.java]: to uncomment soon

**Comment by PashaEagle** [comment]
what`s up bro?

**Comment by Hat1ko** [comment]
> what`s up bro?

Little tough and you?

**Pull request merged**